biggus/__init__.py

```diff
@@ -490,12 +490,16 @@ def __array__(self, dtype=None):
         return np.asarray(result, dtype=dtype)
 
     def __str__(self):
-        fmt = '<Array shape={} dtype={!r} size={}>'
-        return fmt.format(self.shape, self.dtype, size(self))
+        fmt = '<Array shape=({},) dtype={!r} size={}>'
```

Comment:
Though now that I think about it, you can still create an array using a single integer shape, so maybe that's fine.